config/routes.rb

```diff
@@ -41,4 +41,5 @@
   end
 
   get '/auth/:provider/callback', to: 'admin/sessions#create'
+  get '/users', to: 'users#index'
```

Comment:
I don't think this is needed, the user controller takes care of this?

Reply:
yes, this won't work. You can view the users index at /admin/users